CentralManagement/ChocoCCM Command Reference/ExportCCMDeploymentDetail.md

```diff
@@ -0,0 +1,94 @@
+# Export-CCMDeploymentDetail
+
+<!-- This documentation is automatically generated from /Export-CCMDeploymentDetail.ps1 using GenerateDocs.ps1. Contributions are welcome at the original location(s). -->
+
+Downloads a deployment report from Central Management
```

Comment:
Command name should probably match purpose. Is there a reason this is not just `Export-CcmDeploymentReport`?

Reply:
This has been modified, and the help updated to be a bit more descriptive